pkg/models/manifest.go

```diff
@@ -92,6 +92,16 @@ func BootstrapManifests(fluxClient flux.Flux, clusterName string, namespace stri
 		return nil, fmt.Errorf("failed marshalling wego config: %w", err)
 	}
 
+	configBranch, err := gitProvider.GetDefaultBranch(ctx, configURL)
+	if err != nil {
+		return nil, err
+	}
+
+	sourceManifest, err := GetSourceManifest(ctx, fluxClient, gitProvider, clusterName, namespace, configURL, configBranch)
```

Comment:
Does this mean that this manifest isn't being applied on `main`?

Reply:
it is applied but not by using this function (which is the ideal scenario).